### DISCUSSION TITLE
When does the Android Password Store team plan to upload version 2.0.0 to Google Play?

### DISCUSSION THREAD

**Discussion by ghost** [zeapo/Android-Password-Store#2474] [General]
The latest release of Android Password Store on Google Play is version 1.13.5 for almost two years. Athough the newer version, version 2.0.0, includes numerous updates and enhancements, it hasn't been uploaded yet.

When do you plan to upload the newer version as a **stable release** to Google Play?

**Comment by msfjarvis** [comment]
When it is stable :)

There are still [major](https://msfjarvis.dev/aps/issue/1523) [feature](https://msfjarvis.dev/aps/issue/1786) [holes](https://msfjarvis.dev/aps/issue/2017) left to address before we're back to parity with 1.13.x and these are just the PGP parts of it. As a lone developer with very little time and motivation, it's hard to offer a timeline.
